src/plugins/vis_type_tagcloud/public/components/tag_cloud.test.js

```diff
@@ -17,22 +17,29 @@
  * under the License.
  */
 
-import expect from '@kbn/expect';
 import _ from 'lodash';
 import d3 from 'd3';
+import 'jest-canvas-mock';
 
 import { fromNode, delay } from 'bluebird';
-import { ImageComparator } from 'test_utils/image_comparator';
-import simpleloadPng from './simpleload.png';
-
 // Replace with mock when converting to jest tests
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { seedColors } from '../../../../../../plugins/charts/public/services/colors/seed_colors';
-// Will be replaced with new path when tests are moved
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { TagCloud } from '../../../../../../plugins/vis_type_tagcloud/public/components/tag_cloud';
+import { seedColors } from '../../../charts/public/services/colors/seed_colors';
```

Comment:
We don't need the real seed colors here, you can just use a local array of colors (e.g. by inlining the seed colors array). It would be nice to get rid of this deep import into the charts plugin.